test/smoke/generated/graphrbac-data-plane/src/operations/oAuth2PermissionGrantOperations.ts

```diff
@@ -8,13 +8,13 @@
 
 import "@azure/core-paging";
 import { PagedAsyncIterableIterator } from "@azure/core-paging";
-import { OAuth2PermissionGrant } from "../operationsInterfaces";
+import { OAuth2PermissionGrantOperations } from "../operationsInterfaces";
 import * as coreClient from "@azure/core-client";
 import * as Mappers from "../models/mappers";
 import * as Parameters from "../models/parameters";
 import { GraphRbacManagementClientContext } from "../graphRbacManagementClientContext";
 import {
-  OAuth2PermissionGrantDef,
+  OAuth2PermissionGrant,
```

Comment:
@sarangan12 why we have changes like this here, but the smoke test is passing.

Reply:
I don't change anything in the swagger

Reply:
It is possible that there is a change in the swagger in the master branch.